.ci/oci-e2e-deployment.sh

```diff
@@ -103,7 +103,7 @@ export KUBECONFIG_TEST="/tmp/kubeconfig"
 curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/pre-kcp/scripts/provision-openshift-user.sh | bash -s
 export KUBECONFIG="${KUBECONFIG_TEST}"
 
-timeout --foreground 15m "$WORKSPACE"/hack/bootstrap-cluster.sh preview
+timeout --foreground 15m "$WORKSPACE"/hack/bootstrap-cluster.sh preview --keycloak --toolchain
```

Comment:
note that we are using [centralized scripts now](https://github.com/openshift/release/blob/62cafb73ce055b3f29f8a06cee0d6f301821225f/ci-operator/config/redhat-appstudio/infra-deployments/redhat-appstudio-infra-deployments-main.yaml#L31), so changes in this file won't be reflected in the CI job. In fact, the whole `.ci` folder should be [deleted](https://issues.redhat.com/browse/HACBS-789).

Reply:
Good point. I'll remove the changes from here and I'll create separate PRs to update our centralized scripts (this PR shouldn't affect fucntionality of our scripts).